src/styles/_extends.scss

```diff
@@ -75,15 +75,13 @@
 //-- Other --//
 
 %tooltip {
-  .kui-theme--light & {
-    color: $color-text-light-alt;
-    background: $color-bg-light-alt;
-    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.05);
-  }
-
-  .kui-theme--dark & {
-    color: $color-text-dark-alt;
-    background: $color-bg-dark-alt;
-    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.1);
-  }
+  color: var(--color-text-alt);
+  background: var(--color-bg-alt);
+  box-shadow: 0 1px 3px rgba(black, 0.2);
```

Comment:
just a small observation, currently it doesn't look like we have any defined variables for spacing - will adding spacing variables be something that we will explore in the future?

Reply:
I don't think we need to unless they're explicitly reused and we want to be able to batch update them.